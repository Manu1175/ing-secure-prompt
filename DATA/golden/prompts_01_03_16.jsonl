{"id": "prompts_01_03_16:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a short LinkedIn post promoting the 2024 Annual Report with this link: https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf. Keep the tone professional and include a call to read the full report.", "expected_scrub": "Draft a short LinkedIn post promoting the <YEAR> <DOCUMENT_TYPE> with this link: <LINK>. Keep the tone professional and include a call to action.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a one-paragraph investor brief linking to 2024 Pillar 3 Disclosures (https://assets.ing.com/m/5f710bfeff3364d0/original/ING-BE-Pillar-3-Disclosures-Q4-2024.xlsx) and the Additional Pillar III report (https://assets.ing.com/m/42e3b0d86ac55855/original/ING-Belgium-Additional-Pillar-III-Report-2024.pdf).", "expected_scrub": "Prepare a one-paragraph investor brief linking to <YEAR> <DOCUMENT_TYPE_1> (<LINK_1>) and <DOCUMENT_TYPE_2> (<LINK_2>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a newsroom email that indexes two links: 2024 Full Year Results (https://newsroom.ing.be/ing-continues-to-deliver-strong-growth-in-2024-reinforcing-its-commitment-to-customers-and-society) and 2024 Half Year Results (https://newsroom.ing.be/ing-belgium-builds-on-strong-results-with-608-million-profit-after-first-half-of-2024).", "expected_scrub": "Write a newsroom email that indexes two links: <YEAR> <DOC_A> (<LINK_A>) and <YEAR> <DOC_B> (<LINK_B>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create an archive index for 2021 Pillar 3 Disclosures Q1–Q4: Q1 https://assets.ing.com/m/1b95f821bd2bc0cc/original/2021-ING-Belgium-Pillar-III-Disclosures-1Q.xlsx ; Q2 https://assets.ing.com/m/362f3cfe7ce92619/original/2021-ING-Belgium-Pillar-III-Disclosures-2Q.xlsx ; Q3 https://assets.ing.com/m/1cccdb7e18606501/original/2021-ING-Belgium-Pillar-III-Disclosures-3Q.xlsx ; Q4 https://assets.ing.com/m/3e95b1f2f3d00a3d/original/2021-ING-Belgium-Pillar-III-Disclosures-4Q.xlsx.", "expected_scrub": "Create an archive index for <YEAR> <DOCUMENT_TYPE> Q1–Q4: Q1 <LINK_Q1>; Q2 <LINK_Q2>; Q3 <LINK_Q3>; Q4 <LINK_Q4>.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Produce a compliance calendar entry to review Annual Reports (2018–2024) using these links: 2018 https://assets.ing.com/m/34995a3bd3ed078c/original/2018-Annual-report-ING-Belgium-SA-NV.pdf ; 2019 https://assets.ing.com/m/a9c76d878f9bce7/original/2019-Annual-report-ING-Belgium-SANV.pdf ; 2020 https://assets.ing.com/m/705f909b50f25c5a/original/2020-Annual-report-ING-Belgium-SANV.pdf ; 2021 https://assets.ing.com/m/44ec4a2eb627b6e4/original/2021-Annual-report-ING-Belgium-SANV.pdf ; 2022 https://assets.ing.com/m/68cd8d3b05d86ecf/original/2022-Annual-Report-ING-Belgium-SA.pdf ; 2023 http://assets.ing.com/m/38b726ce41627db/original/EN-2023-ING-Belgium-SA-NV-Annual-report.pdf ; 2024 https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf.", "expected_scrub": "Produce a compliance calendar entry to review Annual Reports (<YEAR_RANGE>) using these links: <LINK_LIST>.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a decommission plan for Retired but Operational Production apps: App_46 (IT Custodian Jonathan Perkins; Asset Owner Justin Andersen; Config T82542), App_61 (Karen Gutierrez; Justin Andersen; T13603), App_71 (Sean Taylor; Jeanette Joseph; T65276), App_89 (Joel Reed; Tara Doyle; T19734).", "expected_scrub": "Draft a decommission plan for Retired but Operational Production apps: <APP_LIST> with <IT_CUSTODIAN>/<ASSET_OWNER>/<CONFIG_GROUP>.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Open incidents for Acceptance apps that are Installed but Non-Operational: App_1 (custodian Jimmy Phillips), App_3 (Tyler Fernandez), App_23 (Luis Montoya), App_41 (Edward Jones), App_47 (Thomas Stein).", "expected_scrub": "Open incidents for <ENVIRONMENT> apps that are <STATUS>/<OPERATIONAL_STATUS>: <APP_LIST> (custodians <IT_CUSTODIANS>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Propose a patch window for Acceptance apps that are Installed/Operational: App_63, App_72, App_79, App_88, App_96, App_97.", "expected_scrub": "Propose a patch window for <ENVIRONMENT> apps that are <STATUS>/<OPERATIONAL_STATUS>: <APP_LIST>.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a UAT gate for Acceptance Operational apps: App_17, App_20, App_63, App_72. Include sign-off from custodians (Ashlee Smith, Melissa Navarro, Jonathan Perkins, Trevor Montgomery).", "expected_scrub": "Create a UAT gate for <ENVIRONMENT> <OPERATIONAL_STATUS> apps: <APP_LIST>. Include sign-off from custodians <IT_CUSTODIANS>.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a RACI note consolidating Asset Owner Mary Sims across App_4 (Test/Operational), App_31 (Production/Operational), App_35 (Development/Operational).", "expected_scrub": "Prepare a RACI note consolidating Asset Owner <ASSET_OWNER> across <APP_LIST> (<ENVIRONMENTS>/<OP_STATUS>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a config-group alignment note: App_63 (T80534), App_28 (T45491), App_70 (T19882). Recommend standardizing monitoring and budget tagging by group.", "expected_scrub": "Write a config-group alignment note: <APP_LIST> (<CONFIG_GROUP_LIST>). Recommend standardizing monitoring and budget tagging by group.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a production-readiness checklist for Installed/Operational Production apps: App_31, App_37, App_53, App_95.", "expected_scrub": "Create a production-readiness checklist for <STATUS>/<OPERATIONAL_STATUS> <ENVIRONMENT> apps: <APP_LIST>.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an engagement snapshot for Emily Jones: Mobile Banking (User, Active; last 2025-04-25) and Current Account (Holder, Active; last 2024-11-11).", "expected_scrub": "Write an engagement snapshot for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>; last <LAST_INTERACTION_DATE>) and <PRODUCT_2> (<RELATIONSHIP>, <STATUS>; last <LAST_INTERACTION_DATE>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a cross-sell note for Jane Miller: Life Insurance (Subscriber, Active), Savings Account (Holder, Active), Mortgage (Inactive). Offer a mortgage health check.", "expected_scrub": "Draft a cross-sell note for <FIRST_NAME> <LAST_NAME>: <PRODUCT_A> (<RELATIONSHIP>, <STATUS>), <PRODUCT_B> (<RELATIONSHIP>, <STATUS>), <PRODUCT_C> (<RELATIONSHIP>, <STATUS>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose gentle nudges to re-engage Online Banking users now Inactive: John Taylor (last 2025-03-19) and Katie Taylor (last 2025-05-04).", "expected_scrub": "Compose gentle nudges to re-engage <PRODUCT> users now <STATUS>: <FIRST_NAME_1> <LAST_NAME_1> (last <DATE_1>) and <FIRST_NAME_2> <LAST_NAME_2> (last <DATE_2>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a relationship matrix for Alex Johnson: Mobile Banking (User, Active), Mortgage (Borrower, Active), Personal Loan (Borrower, Active).", "expected_scrub": "Create a relationship matrix for <FIRST_NAME> <LAST_NAME>: <PRODUCT_1> (<RELATIONSHIP>, <STATUS>), <PRODUCT_2> (<RELATIONSHIP>, <STATUS>), <PRODUCT_3> (<RELATIONSHIP>, <STATUS>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Produce a microsegment for Mortgage borrowers (Active): John Lewis (last 2025-09-09), Jane Smith (last 2025-06-08), Tom Smith (last 2025-01-20). Suggest a rate alert campaign.", "expected_scrub": "Produce a microsegment for <PRODUCT> borrowers (<STATUS>): <FIRST_LAST_1> (last <DATE_1>), <FIRST_LAST_2> (last <DATE_2>), <FIRST_LAST_3> (last <DATE_3>). Suggest a campaign.", "entities": null, "notes": null}
{"id": "prompts_01_03_16:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a Mobile Banking (Active) microsegment with last interactions: Alex Williams (2025-03-02), Emily Jones (2025-04-25), Robert Williams (2025-08-08).", "expected_scrub": "Draft a <PRODUCT> (<STATUS>) microsegment with last interactions: <FIRST_LAST_1> (<DATE_1>), <FIRST_LAST_2> (<DATE_2>), <FIRST_LAST_3> (<DATE_3>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a consent & preferences audit note for Emily Smith: Online Banking (User, Active; last 2025-08-07), Investment Portfolio (Investor, Active; last 2025-07-09), Personal Loan (Borrower, Active; last 2024-10-19).", "expected_scrub": "Compose a consent & preferences audit note for <FIRST_NAME> <LAST_NAME>: <PRODUCT_A> (<RELATIONSHIP>, <STATUS>; last <DATE_A>), <PRODUCT_B> (<RELATIONSHIP>, <STATUS>; last <DATE_B>), <PRODUCT_C> (<RELATIONSHIP>, <STATUS>; last <DATE_C>).", "entities": null, "notes": null}
{"id": "prompts_01_03_16:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_01_03_16.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a data-minimization note: for customer analytics exports referencing names like Jane Miller or John Taylor, use pseudonymous IDs and restrict raw-name access to least-privileged roles.", "expected_scrub": "Create a data-minimization note: for customer analytics exports referencing names like <EXAMPLE_NAME_1> or <EXAMPLE_NAME_2>, use pseudonymous IDs and restrict raw-name access to least-privileged roles.", "entities": null, "notes": null}
