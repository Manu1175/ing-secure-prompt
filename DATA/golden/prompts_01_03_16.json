[
  {
    "text": "Draft a short LinkedIn post promoting the 2024 Annual Report with this link: https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf. Keep the tone professional and include a call to read the full report.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Prepare a one-paragraph investor brief linking to 2024 Pillar 3 Disclosures (https://assets.ing.com/m/5f710bfeff3364d0/original/ING-BE-Pillar-3-Disclosures-Q4-2024.xlsx) and the Additional Pillar III report (https://assets.ing.com/m/42e3b0d86ac55855/original/ING-Belgium-Additional-Pillar-III-Report-2024.pdf).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write a newsroom email that indexes two links: 2024 Full Year Results (https://newsroom.ing.be/ing-continues-to-deliver-strong-growth-in-2024-reinforcing-its-commitment-to-customers-and-society) and 2024 Half Year Results (https://newsroom.ing.be/ing-belgium-builds-on-strong-results-with-608-million-profit-after-first-half-of-2024).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create an archive index for 2021 Pillar 3 Disclosures Q1–Q4: Q1 https://assets.ing.com/m/1b95f821bd2bc0cc/original/2021-ING-Belgium-Pillar-III-Disclosures-1Q.xlsx ; Q2 https://assets.ing.com/m/362f3cfe7ce92619/original/2021-ING-Belgium-Pillar-III-Disclosures-2Q.xlsx ; Q3 https://assets.ing.com/m/1cccdb7e18606501/original/2021-ING-Belgium-Pillar-III-Disclosures-3Q.xlsx ; Q4 https://assets.ing.com/m/3e95b1f2f3d00a3d/original/2021-ING-Belgium-Pillar-III-Disclosures-4Q.xlsx.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Produce a compliance calendar entry to review Annual Reports (2018–2024) using these links: 2018 https://assets.ing.com/m/34995a3bd3ed078c/original/2018-Annual-report-ING-Belgium-SA-NV.pdf ; 2019 https://assets.ing.com/m/a9c76d878f9bce7/original/2019-Annual-report-ING-Belgium-SANV.pdf ; 2020 https://assets.ing.com/m/705f909b50f25c5a/original/2020-Annual-report-ING-Belgium-SANV.pdf ; 2021 https://assets.ing.com/m/44ec4a2eb627b6e4/original/2021-Annual-report-ING-Belgium-SANV.pdf ; 2022 https://assets.ing.com/m/68cd8d3b05d86ecf/original/2022-Annual-Report-ING-Belgium-SA.pdf ; 2023 http://assets.ing.com/m/38b726ce41627db/original/EN-2023-ING-Belgium-SA-NV-Annual-report.pdf ; 2024 https://assets.ing.com/m/6e8ace8ade094690/original/Annual-report-2024.pdf.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft a decommission plan for Retired but Operational Production apps: App_46 (IT Custodian Jonathan Perkins; Asset Owner Justin Andersen; Config T82542), App_61 (Karen Gutierrez; Justin Andersen; T13603), App_71 (Sean Taylor; Jeanette Joseph; T65276), App_89 (Joel Reed; Tara Doyle; T19734).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Open incidents for Acceptance apps that are Installed but Non-Operational: App_1 (custodian Jimmy Phillips), App_3 (Tyler Fernandez), App_23 (Luis Montoya), App_41 (Edward Jones), App_47 (Thomas Stein).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Propose a patch window for Acceptance apps that are Installed/Operational: App_63, App_72, App_79, App_88, App_96, App_97.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a UAT gate for Acceptance Operational apps: App_17, App_20, App_63, App_72. Include sign-off from custodians (Ashlee Smith, Melissa Navarro, Jonathan Perkins, Trevor Montgomery).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Prepare a RACI note consolidating Asset Owner Mary Sims across App_4 (Test/Operational), App_31 (Production/Operational), App_35 (Development/Operational).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write a config-group alignment note: App_63 (T80534), App_28 (T45491), App_70 (T19882). Recommend standardizing monitoring and budget tagging by group.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a production-readiness checklist for Installed/Operational Production apps: App_31, App_37, App_53, App_95.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write an engagement snapshot for Emily Jones: Mobile Banking (User, Active; last 2025-04-25) and Current Account (Holder, Active; last 2024-11-11).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Draft a cross-sell note for Jane Miller: Life Insurance (Subscriber, Active), Savings Account (Holder, Active), Mortgage (Inactive). Offer a mortgage health check.",
    "labels": []
  },
  {
    "text": "Compose gentle nudges to re-engage Online Banking users now Inactive: John Taylor (last 2025-03-19) and Katie Taylor (last 2025-05-04).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Create a relationship matrix for Alex Johnson: Mobile Banking (User, Active), Mortgage (Borrower, Active), Personal Loan (Borrower, Active).",
    "labels": []
  },
  {
    "text": "Produce a microsegment for Mortgage borrowers (Active): John Lewis (last 2025-09-09), Jane Smith (last 2025-06-08), Tom Smith (last 2025-01-20). Suggest a rate alert campaign.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Draft a Mobile Banking (Active) microsegment with last interactions: Alex Williams (2025-03-02), Emily Jones (2025-04-25), Robert Williams (2025-08-08).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Compose a consent & preferences audit note for Emily Smith: Online Banking (User, Active; last 2025-08-07), Investment Portfolio (Investor, Active; last 2025-07-09), Personal Loan (Borrower, Active; last 2024-10-19).",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Create a data-minimization note: for customer analytics exports referencing names like Jane Miller or John Taylor, use pseudonymous IDs and restrict raw-name access to least-privileged roles.",
    "labels": []
  }
]